forta-uniswap-event-py/src/agent.py

```diff
@@ -0,0 +1,216 @@
+import binascii
+import json
+
+from forta_agent import Finding, FindingType, FindingSeverity
```

Comment:
Have we codified our convention on importing into the global namespace?
These agents will likely import very few modules and functions, but my preference is to import modules and call the functions with `<module>.<function>` rather than importing functions directly into the global namespace, when possible.

Reply:
We haven't codified it, both are good ways of importing modules. The Forta examples from the SDK developers use this convention so I think I'll stick with it here.